lib/video/tasks.py

```diff
@@ -85,12 +71,7 @@ def _resize_video(src, instance, **kw):
             os.makedirs(dirs)
 
     shutil.move(thumbnail_file, instance.thumbnail_path)
-    if waffle.switch_is_active('video-encode'):
-        # Move the file over, removing the temp file.
-        shutil.move(video_file, instance.image_path)
-    else:
-        # We didn't re-encode the file.
```

Comment:
it means the if statement didn't happen, i'm not sure it was helpful the first time around